nservicebus/hosting/critical-errors.md

```diff
@@ -46,12 +46,22 @@ snippet:CustomHostErrorHandlingAction
 
 ## When to override the default action
 
-The default action should be overridden whenever that default does not meet the specific hosting requirements. For example
+When self hosting the critical error callback must be overriden in order to exit the current process. Not overriding the critical error callback will result in the endpoint instance to stop processing messages without the ability to recover from this state.
```

Comment:
can this be reformulated into something like "the default behavior is to stop the endpoint, not the process. When self hosting endpoints, it's recommended override the CE action to exit the process"?